content/guides/references/configuration.md

```diff
@@ -191,18 +191,17 @@ The Node version is used in Cypress to:
 - Execute code in the
   [pluginsFile](/guides/references/configuration#Folders-Files).
 
-<DocsImage src="/img/guides/test-runner-settings-nodejs-version.jpg" alt="Node version in Settings in Test Runner" ></DocsImage>
+<DocsImage src="/img/guides/test-runner-settings-nodejs-version.jpg" alt="Node version in Settings in Cypress App" ></DocsImage>
 
 ### Experiments
 
 Configuration might include experimental options currently being tested. See
 [Experiments](/guides/references/experiments) page.
 
-## Runner Specific Options
```

Comment:
Anywhere you've added the text "Test Type" can you use "Testing Type" instead? This mirrors the `Cypress.testingType` property.